sklearn/datasets/lfw.py

```diff
@@ -77,20 +77,13 @@
 )
 
 
-def scale_face(face):
```

Comment:
If this is public it can't be simply removed.

Reply:
Same as above: it's not exported in `datasets/__init__.py` so I figured it can't be used anyway. Maybe I'm wrong?

Reply:
hm... I guess I'd be fine with it, let's see what the others say.

Reply:
Looks like `scale_face` was introduced in 6826401dc6e3390c747d83cde9fe0a5263feca3c and was used in  [examples/applications/plot_face_verification.py](https://github.com/scikit-learn/scikit-learn/blob/master/examples/applications/plot_face_recognition.py), but it's not the case anymore.

Reply:
I think we should probably not make it private in this case. But we can deprecate it being public.

Reply:
I put it back and added a docstring